src/parser/rogue/assassination/modules/features/RuptureSnapshot.ts

```diff
@@ -13,25 +13,25 @@ const COMBO_POINT_DURATION = 4000;
  * Identify inefficient refreshes of the Rupture DoT.
  */
 class RuptureSnapshot extends Snapshot {
-  get durationOfFresh() {
-    return BASE_DURATION + this.comboPointsOnLastCast * COMBO_POINT_DURATION;
-  }
-
   static spellCastId = SPELLS.RUPTURE.id;
   static debuffId = SPELLS.RUPTURE.id;
   static spellIcon = SPELLS.RUPTURE.icon;
   comboPointsOnLastCast = 0;
 
-  constructor(...args) {
-    super(...args);
+  constructor(options: Options) {
+    super(options);
     const combatant = this.selectedCombatant;
     if (combatant.hasTalent(SPELLS.SUBTERFUGE_TALENT.id)) {
       this.active = false;
     }
     this.addEventListener(Events.SpendResource.by(SELECTED_PLAYER).spell(SPELLS.RUPTURE), this.onSpendResource);
```

Comment:
You should make this prettier so it'd read this.active = this.selectedCombatant.hasTalent(SPELLS.SUBTERFUGE_TALENT.id); 
Rest is pointless when you don't use the combatant object elsewhere.